osmnx/utils.py

```diff
@@ -477,7 +477,7 @@ def get_nearest_node(G, point, method='haversine', return_dist=False):
         raise ValueError('method argument must be either "haversine" or "euclidean"')
 
     # nearest node's ID is the index label of the minimum distance
-    nearest_node = int(distances.idxmin())
+    nearest_node = distances.idxmin()
```

Comment:
Will this invariably return the min index label as an int when working with OSM data? This was originally here because under certain circumstances it would return a float version of the label.

Reply:
This was introduced by the array above, see: `np.array([1,  .2])`

Reply:
Ah right. Thanks.